mlflow/recipes/steps/evaluate.py

```diff
@@ -61,19 +63,16 @@ def _validate_and_apply_step_config(self):
                 "Missing recipe config in recipe config.",
                 error_code=INVALID_PARAMETER_VALUE,
             )
-        if "positive_class" not in self.step_config and self.recipe == "classification/v1":
-            raise MlflowException(
-                "`positive_class` must be specified for classification/v1 recipes.",
-                error_code=INVALID_PARAMETER_VALUE,
-            )
```

Comment:
Yes, it's not an error anymore. If the user does not define the positive_class parameter, the problem will be treated as a multi-class problem. We can add another parameter to let the user explicitly decide if he is dealing with a binary or multi-class classification problem instead of using the presence of a positive_class parameter to decide whether it's a binary or multi-class.  Does it make sense? @mengxr Is it ok to use positive_class for this decision?

Reply:
Even in a multi-class scenario, I can still define a "positive" class. Given the fact that we can detect whether it's a binary class or multi-class problem easily, I suggest to keep this warning for binary class problem.

Reply:
Ok, that makes sense. I have returned the warning for binary classification.